src/test/java/de/mediathekview/mserver/crawler/CrawlerManagerFTPTest.java

```diff
@@ -1,11 +1,21 @@
 package de.mediathekview.mserver.crawler;
 
+import de.mediathekview.mlib.filmlisten.FilmlistFormats;
+import de.mediathekview.mlib.messages.Message;
+import de.mediathekview.mlib.messages.MessageTypes;
+import de.mediathekview.mlib.messages.MessageUtil;
+import de.mediathekview.mlib.messages.listener.MessageListener;
+import de.mediathekview.mlib.progress.Progress;
+import de.mediathekview.mlib.progress.ProgressListener;
+import de.mediathekview.mserver.base.config.MServerConfigDTO;
+import de.mediathekview.mserver.base.config.MServerConfigManager;
+import de.mediathekview.mserver.base.config.MServerFTPSettings;
+import de.mediathekview.mserver.testhelper.FileReader;
 import java.io.File;
 import java.io.IOException;
 import java.net.URISyntaxException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.net.URISyntaxException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)